lib/rules/sort-comp.js

```diff
@@ -128,20 +128,30 @@ module.exports = {
       const indexes = [];
 
       if (method.static) {
-        for (i = 0, j = methodsOrder.length; i < j; i++) {
-          if (methodsOrder[i] === 'static-methods') {
-            indexes.push(i);
-            break;
-          }
+        const staticIndex = methodsOrder.indexOf('static-methods');
+        if (staticIndex >= 0) {
```

Comment:
Thanks for cleaning these others up too :-) I didn't realize you were following an existing pattern.